### ISSUE TITLE
polled_camera: include files are installed to the wrong location

### ISSUE THREAD

**Issue by wjwwood** [ros-perception/image_common#2]
On the latest ros-shadow-fixed deb for polled_camera, the headers are installed to an incorrect location:

```
∫ ls /opt/ros/groovy/include/polled_camera/polled_camera/
publication_server.h
```

This causes prosilica_camera to fail.

**Comment by vrabaud** [comment]
fixed in 8a4fd3d86054ff66f0aee8017e0a98de5daa70ef
Releasing.

**Comment by wjwwood** [comment]
Thanks.

On Fri, Oct 12, 2012 at 4:09 PM, Vincent Rabaud notifications@github.comwrote:

> fixed in 8a4fd3dhttps://github.com/ros-perception/image_common/commit/8a4fd3d86054ff66f0aee8017e0a98de5daa70ef
> Releasing.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ros-perception/image_common/issues/2#issuecomment-9396593.

## 

William Woodall
Willow Garage - Software Engineer
wwoodall@willowgarage.com
